### PR TITLE
chore(ci): adding the go linter

### PR DESCRIPTION
**Pull request by rdefosse**
Testing from sub-repo

### CODE REVIEW

File: gnodeb/worker/gnbamfworker/handler.go [hunk 1]
```diff
@@ -31,6 +30,7 @@ func HandleNgSetupResponse(amf *gnbctx.GnbAmf, pdu *ngapType.NGAPPDU) {
 		amf.Log.Errorln("ran is nil")
 		return
 	}
+	amf.Log.Traceln("Processing NG Setup Response")
```
Comment (lionelgo) [inline gnodeb/worker/gnbamfworker/handler.go:33]: why do you put trace here, meaning not before, not after parameters checking ?

Comment (rdefosse) [reply gnodeb/worker/gnbamfworker/handler.go:33]: The code was previously:

```go
amf.Log.Traceln("Processing NG Setup Response")
if (amf == nil) {
    amf.Log.Errorln("ran is nil")
    return
}
```

The linter was rightfully complaining that we could be using a null pointer.
So I moved the log line after the check.

Comment (rdefosse) [reply gnodeb/worker/gnbamfworker/handler.go:33]: In fact it does not complain about the `amf.log.Errorln` within the braces.
How weird.

Comment (rdefosse) [reply gnodeb/worker/gnbamfworker/handler.go:33]: Something like this:

```
        if amf == nil {
-               amf.Log.Errorln("ran is nil")
+               err := fmt.Errorf("amf pointer is nil")
+               fmt.Println(err)
                return
        }
```

What do you think?

Comment (lionelgo) [reply gnodeb/worker/gnbamfworker/handler.go:33]: Fine for nil amf, have to check loggers versus println...

Comment (lionelgo) [reply gnodeb/worker/gnbamfworker/handler.go:36]: well, a default logger is missing in this app

Comment (lionelgo) [reply gnodeb/worker/gnbamfworker/handler.go:36]: yes, can be a workaround

**Review by lionelgo — APPROVED** [review]

**Pull request merged**